app/jobs/memberships/send_membership_request_mailer_job.rb

```diff
@@ -60,11 +60,12 @@ def external_message_id(msg, mailer_parameters)
 
   def email_nil?(email)
     if email.nil?
```

Comment:
```suggestion
    email.try(:to).nil?
```

It looks like we may be wanting to check specifically for the presence of the "to" address in an `ActionMailer::Parameterized::MessageDelivery` object. If so, this change should allow for that.